src/application-manager/application-manager.ts

```diff
@@ -119,16 +94,16 @@ export class ApplicationManager {
 
   initLocalApps(clearStore: boolean = false): Promise<void[]> {
     return new Promise((resolve, reject) => {
-      fs.readdir(join(app.getAppPath(), 'build', 'applications'), (err, files) => {
+      // TODO: open ticket @ electron "withFileTypes" option is not working inside of the asar archive, it only returns the file names
```

Comment:
What about we create a ticket with this task 😊

Reply:
Actually already done by other peoples: https://github.com/electron/electron/issues/25349

Also cf https://github.com/electron/electron/pull/24062